### PR TITLE
Change default format for ecosystem checks to `markdown`

### PR DESCRIPTION
**Pull request by zanieb**
To facilitate easier local runs

### CODE REVIEW

File: python/ruff-ecosystem/README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ Run `ruff format` ecosystem checks comparing your debug build to your system Ruf
 ruff-ecosystem format ruff "./target/debug/ruff"
 ```
 
+The default output format is markdown, but you can use `--output-format json` to get raw data — this is
```
Comment (charliermarsh) [inline python/ruff-ecosystem/README.md:34]: Which one is particularly useful when developing? Markdown or JSON?

**Review by charliermarsh — APPROVED** [review]

File: python/ruff-ecosystem/ruff_ecosystem/cli.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--output-format",
         choices=[option.name for option in OutputFormat],
-        default="json",
+        default="markdown",
```
Comment (charliermarsh) [inline python/ruff-ecosystem/ruff_ecosystem/cli.py:124]: Do we need to pass this in somewhere now?

Comment (zanieb) [reply python/ruff-ecosystem/ruff_ecosystem/cli.py:124]: No in CI we pass `--output-format markdown` already

File: python/ruff-ecosystem/README.md (hunk 1 above)
Comment (zanieb) [inline python/ruff-ecosystem/README.md:34]:
```suggestion
The default output format is markdown, which includes nice summaries of the changes. You can use `--output-format json` to display the raw data — this is
```

File: python/ruff-ecosystem/README.md [hunk 3]
```diff
@@ -31,6 +31,9 @@ Run `ruff format` ecosystem checks comparing your debug build to your system Ruf
...
+particularly useful when developing.
```
Comment (zanieb) [inline python/ruff-ecosystem/README.md:35]:
```suggestion
particularly useful when making changes to the ecosystem checks. 
```

**Pull request merged**